### ISSUE TITLE
Wintergrasp Graveyard Immunity Buff bug

### ISSUE THREAD

**Issue by E425** [WoWManiaUK/Redemption#1611]
The immunity buff at the wintergrasp graveyards does not prevent damage from sieges, demos, or cats. The immunity buff should prevent ALL damage, even from vehicles.

**Comment by fvicaria** [comment]
Immunity is from other players olnly.

**Comment by E425** [comment]
I respectfully disagree.... why have an immunity buff at all then? Opposing faction just attacks players at gy with sieges. It makes players loose interest in pvp. I'm pretty sure the immunity buff in retail wotlk worked for sieges.
